website/src/contribute/release-guide.md

```diff
@@ -699,7 +711,18 @@ done
 * Build Java images and push to DockerHub.
 
 ```
-./gradlew :sdks:java:container:dockerPush -Pdocker-pull-licenses -Pdocker-tag=${RELEASE}_rc{RC_NUM}
+./gradlew :sdks:java:container:docker -Pdocker-pull-licenses -Pdocker-tag=${RELEASE}_rc{RC_NUM}
+```
+
+Verify that third party licenses are included by logging in to the images. For Java SDK images, there should be around 1400 dependencies. 
+```
+docker run -it --entrypoint=/bin/bash apache/beam_java_sdk:${RELEASE}_rc{RC_NUM}
+ls -al /opt/apache/beam/third_party_licenses/ | wc -l
+```
+
+After verifying the third party licenses are included correctly, push the images to DockerHub.
+```
```

Comment:
@ibzib , this part changed slightly. Please note the change from `dockerPush` to `docker` at L714.
After creating the Java image, we should verify it BEFORE pushing to DockerHub. I'm not sure when this PR can be merged because of website issues, so explicitly pinging you here.